source/_data/themes.yml

```diff
@@ -2393,7 +2393,7 @@
     - responsive
 - name: Hive
   description: a minimalist theme for hexo
-  link: https://github.com/farnaziifz/hexo-hive
+  link: https://github.com/farnaziifz/hive-hexo
   preview: http://51.68.177.152/
```

Comment:
IMHO specify ip address is not good some point of view.
In fact, other theme owners specify themselves domain or github.io.

I think use github pages is better and fastest way for publish static site, if you hasn't your domain.

* [GitHub pages](https://pages.github.com/)
* [Publishing github pages](https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/)

Reply:
Done !!!